lib/Test/Nginx/Socket.pm

```diff
@@ -2864,6 +2873,7 @@ The following sections are supported:
 =head2 http2
 
 Enforces the test scaffold to use the HTTP/2 wire protocol to send the test request.
+Also, you can set tls section using the HTTP/2 over TLS protocol and SNI(Server Name Indication).
```

Comment:
Do we really need to mention SNI here? It can be implicit I think. The user cannot specify their own SNI names now anyway.